script.js

```diff
@@ -193,6 +193,26 @@ function updateStaffMember(mID) {
     request.send(form);
 }
 
+// block staff user
+function blockUnblockUser(mID, statusID) {
+    var mID = mID;
```

Comment:
This variable assign operation is useless. You can directly use mID, statusID local variables.